CONTRIBUTING.md

```diff
@@ -1,7 +1,66 @@
 # Contributing to the infrastructure template
 
+## Getting started
+
 If you are looking to contribute, get started by reading the following docs.
 
 - Read about the [infrastructure's module architecture](/docs/infra/module-architecture.md) to learn how the architecture of the infrastructure code is designed and how the modules interact with each other.
 - Read the [template development workflow](/template-only-docs/template-development-workflow.md) to understand how to develop and test changes to the template because working on the platform templates is unlike working on most other applications.
 - Read the [infrastructure style guide](/docs/infra/style-guide.md) to understand best practices for Terraform and shell scripts.
+
+## Isolate PRs to a single infrastructure layer and make each individual PR backwards compatible
+
+Each PR should only touch one of the infrastructure layers at a time. The infrastructure layers are:
+
+- Accounts layer (infra/accounts)
```

Comment:
Rather than re-listing, should maybe point to `/infra/README.md#infrastructure-layers` as the source of truth at the moment.